client/src/components/Leaderboard.test.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import LeaderBoard from "./Leaderboard";
+import renderer from "react-test-renderer";
+
+it("renders without crashing", () => {
+  const tree = renderer.create(<LeaderBoard />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
A simple frontend unit test.

As our auth is broken and a ton of other issues, there was a lot of problems running tests in the frontend. I had to settle for a single simple test. 

This also gives me credence that we need to get better with our testing. Had we had testing, I think that we would not have had such an issue with auth. 

(sorry for the opinion piece. Let's work on writing more tests)